OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -188,8 +188,14 @@ public final View getView(int position, View convertView, ViewGroup parent) {
                     final Activity activity = parentFragment.getActivity();
                     if (activity != null && ((RoboAppCompatActivity) activity).isInForeground()) {
                         final BulkDownloadFragment fragment = new BulkDownloadFragment(downloadListener, environment);
-                        parentFragment.getChildFragmentManager().
-                                beginTransaction().replace(convertView.getId(), fragment).commit();
+                        final View finalConvertView = convertView;
+                        // Wait until the convertView has attached with the parent view.
+                        // Use commitNowAllowingStateLoss() method, there is a chance that
```

Comment:
`Using commitNowAllowingStateLoss() method here because there is chance transaction could have happened even the fragments state is saved.`
is a better comment